src/scarabee/_scarabee/python/cmfd.cpp

```diff
@@ -0,0 +1,183 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/eigen.h>
+
+#include <moc/cmfd.hpp>
+
+namespace py = pybind11;
+
+using namespace scarabee;
+
+void init_CMFD(py::module& m) {
+
+  py::enum_<CMFDSurfaceCrossing::Type>(m,"CMFDSurfaceCrossingType")
+    .value("XN",CMFDSurfaceCrossing::Type::XN)
+    .value("XP",CMFDSurfaceCrossing::Type::XP)
+    .value("YN",CMFDSurfaceCrossing::Type::YN)
+    .value("YP",CMFDSurfaceCrossing::Type::YP)
+    .value("TR",CMFDSurfaceCrossing::Type::TR)
+    .value("BR",CMFDSurfaceCrossing::Type::BR)
+    .value("BL",CMFDSurfaceCrossing::Type::BL)
+    .value("TL",CMFDSurfaceCrossing::Type::TL);
```

Comment:
These should all have description doc strings. Please check in other places to see how I add those for enum types.

Reply:
That is because I am a lazy potato. The `DiffusionData` bindings have examples though ! Actually, you should directly copy some of those doc strings here.